code/modules/mob/living/carbon/brain/posibrain.dm

```diff
@@ -28,6 +29,13 @@
 				if(check_observer(O))
 					transfer_personality(O)
 			reset_search()
+	else
+		if(silenced)
+			silenced = 0
+			user << "<span class='notice'>You toggle the speaker to 'on', on the [src].</span>"
+		else
+			silenced = 1
+			user << "<span class='notice'>You toggle the speaker to 'off', on the [src].</span>"
 
```

Comment:
If you want this to only be called on active posibrains, you can check for `brainmob && brainmob.key`